internal/api/auth.go

```diff
@@ -129,3 +134,22 @@ func BuildJWT(user int) (string, error) {
 	// возвращаем строку токена
 	return tokenString, nil
 }
+
+func GetUserID(tokenString string) (int, error) {
```

Comment:
Тот случай, когда приемлемо использовать указатель на `*int` для того чтобы можно было вернуть `nil` в качестве отсутствующего значения. 
```suggestion
func getUserID(tokenString string) (*int, error) {
```